CHANGELOG.md

```diff
@@ -22,7 +22,7 @@
   * Fixed support in the case of projects with spaces in path. [aspnetcore#23336](https://github.com/dotnet/aspnetcore/issues/23336)
   * Various performance improvements.
   * `@inject` completion [aspnetcore#22886](https://github.com/dotnet/aspnetcore/issues/22886)
-  * Improved behavior in cases where expected files are not available. [dotnet/aspnetcore-tooling#2008](https://github.com/dotnet/aspnetcore-tooling/pull/2008)
+  * Improved behavior in cases where directory is not available. [dotnet/aspnetcore-tooling#2008](https://github.com/dotnet/aspnetcore-tooling/pull/2008)
```

Comment:
Another one is we actually added an "Extract to CodeBehind" light bulb 😄

Reply:
fyi/ @noahbkim